modules/openapi-generator/src/main/resources/cpp-restbed-server/api-header.mustache

```diff
@@ -49,6 +50,31 @@ public:
     {{#vendorExtensions.x-codegen-otherMethods}}
     void {{httpMethod}}_method_handler(const std::shared_ptr<restbed::Session> session);
     {{/vendorExtensions.x-codegen-otherMethods}}
+
+	void set_callback_{{httpMethod}}(
```

Comment:
If there are multiple endpoints with the same tag in the API returning and/or accepting similar parameters this will cause  redefinition of the methods

Reply:
Could you provide me an example when this could happen?
I tried something like this:

```
paths:
  '/method/{param1}':
    parameters:
      - name: param1
        in: path
        type: string
        required: true
        description: Some string parameter
    post:
      operationId: methodParam1Post
      tags:
        - sometag
    (...)
  '/method/{param2}':
    parameters:
      - name: param2
        in: path
        type: integer
        required: true
        description: Some other parameter
    post:
      operationId: methodParam2Post
      tags:
        - sometag
    (...)
```

but I got this:
```
class  SometagApiMethodParam1Resource: public restbed::Resource
{
public:
	SometagApiMethodParam1Resource();
    virtual ~SometagApiMethodParam1Resource();
    void POST_method_handler(const std::shared_ptr<restbed::Session> session);

	void set_handler_POST(
		std::function<std::pair<int, std::string>(
			std::string const &
		)> handler
	);

private:
	std::function<std::pair<int, std::string>(
		std::string const &
	)> handler_POST_;
};

class  SometagApiMethodParam2Resource: public restbed::Resource
{
public:
	SometagApiMethodParam2Resource();
    virtual ~SometagApiMethodParam2Resource();
    void POST_method_handler(const std::shared_ptr<restbed::Session> session);

	void set_handler_POST(
		std::function<std::pair<int, std::string>(
			int32_t const &
		)> handler
	);


private:
	std::function<std::pair<int, std::string>(
		int32_t const &
	)> handler_POST_;

};
```

Reply:
I see you are right, I was thinking in the terms of the other framework. It looks fine then.